### ISSUE TITLE
Don't change the font type, the old one was absolutely great

### ISSUE THREAD

**Issue by navakelvin** [tldraw/tldraw-beta#39]
The new font type is horrible, guys.
## New
![image](https://user-images.githubusercontent.com/77164639/212409024-5091d996-3597-4b2d-80e7-ca4df7e1c9b5.png)

Please, consider leaving the previous one, it is beautiful and the condensation allows the text to not be spread all over the screen.
## Old
![image](https://user-images.githubusercontent.com/77164639/212409395-a8c79960-e8b2-4dfb-bd92-703f5db80881.png)

PS. Or consider leaving them both perhaps.

**Comment by TodePond** [comment]
Thanks for your honest feedback!

Just so I understand, when you say 'spread all over the screen', do you mean that the new font takes up too much horizontal space?

**Comment by navakelvin** [comment]
Exactly, it takes up too much horizontal space :)

**Comment by webjoyable** [comment]
I strongly disagree, one of the reasons I use beta is because of the font and the UI. The new font (in beta) is way better and more consistent with the layout of the drawing elements

**Comment by navakelvin** [comment]
It is reasonable that there are different preferences.

However, someone else will probably agree that the beta source has a lower height. Therefore you should enlarge the text, making it take up more space horizontally.

It's not a tracking or kerning issue, it's a height issue.

**Comment by webjoyable** [comment]
> It is reasonable that there are different preferences.
> 
> However, someone else will probably agree that the beta source has a lower height. Therefore you should enlarge the text, making it take up more space horizontally.
> 
> It's not a tracking or kerning issue, it's a height issue.
> 
> I wouldn't mind the current font remaining. But I honestly think a pinch more height would be better.

I agree that there are different preferences in the compactness of the content, but I don't think that developers chose this font on a whim, it must have satisfied certain specifications. On the other hand, It would be awesome if we could choose between several fonts (not too many 2 or 3 because the font is part of the brand), there is already an option to change font type, so probably wouldn't be that difficult to accommodate

**Comment by TodePond** [comment]
After using the new font for a while now, I can safely say that I really like how it matches the overall tldraw style :)

It's useful to hear feedback about it though.

I imagine that many people who build with tldraw might want to change the font too, depending on their use-case!
